sbnana/CAFAna/Systs/NuMIFluxSysts.cxx

```diff
@@ -84,13 +85,15 @@ namespace ana
 
     if(bin == 0 || bin == h->GetNbinsX()+1) return;
 
-    weight *= 1 + sigma*h->GetBinContent(bin);
+    // BH -- attempt to not change any weight if the value is inf or nan, just continue with weight as-is...
+    if( std::isinf(h->GetBinContent(bin)) || std::isnan(h->GetBinContent(bin)) ) weight *= 1;
+    else weight *= 1 + sigma*h->GetBinContent(bin);
```

Comment:
I guess this is something I added when I was tinkering with these files in my working area? Was I right to add some protection here: are there potential infs/nans to worry about in the weights?